src/storage/store/node_table.cpp

```diff
@@ -43,6 +43,18 @@ void NodeTable::read(Transaction* transaction, TableReadState& readState, ValueV
     }
 }
 
+common::offset_t NodeTable::validatePkConstraint(
```

Comment:
I think `validateUniquenessConstraint` is more accurate, as PK also has non null constraint, which is not checked here.